cmd/root.go

```diff
@@ -124,8 +127,35 @@ func modifyToSentryLogger(log *zap.Logger, client *sentry.Client) *zap.Logger {
 	}
 
 	log = zapsentry.AttachCoreToLogger(core, log)
-
-	return log.With(zapsentry.NewScope())
+	kernelVersion := "Not linux"
```

Comment:
What is the significance of this value..?

Reply:
reverted